src/listener/concurrent_listener.rs

```diff
@@ -113,8 +113,7 @@ where
     fn info(&self) -> Vec<ListenInfo> {
         self.listeners
             .iter()
-            .map(|listener| listener.info().into_iter())
-            .flatten()
+            .flat_map(|listener| listener.info().into_iter())
```

Comment:
This isn't related to the wasm change, but failed [CI](https://github.com/http-rs/tide/runs/5529391825?check_suite_focus=true) for clippy